package.json

```diff
@@ -158,6 +158,7 @@
         "lit-analyzer": "^1.2.1",
         "mocha-junit-reporter": "^2.0.2",
         "netlify-cli": "^12.9.2",
+        "next": "^13.4.1",
```

Comment:
Does out project need this dependency to work? None of our tooling ever _runs_ the React components, so it would be great not to add the underlying packages here to out lock...

Reply:
It was required for the typescript compiling. I tried to avoid it like I did for the react package, but unfortunately Next.js doesn't provide a type definition package like react did.